### PR TITLE
Update solution for use-breadth-first-search-in-a-binary-search-tree.english.md

### PR DESCRIPTION
**Pull request by sagarkharabe**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/use-breadth-first-search-in-a-binary-search-tree.english.md [hunk 1]
```diff
@@ -106,7 +106,42 @@ BinarySearchTree.prototype = Object.assign(
 <section id='solution'>
 
 ```js
-// solution required
+var displayTree = tree => console.log(JSON.stringify(tree, null, 2));
```
Comment (moT01) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/use-breadth-first-search-in-a-binary-search-tree.english.md:109]:
```suggestion
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Probably don't need that in there? Other than that, looks good to me.

**Comment by moT01** [comment]
I already added this to the guide.

**Review by moT01 — APPROVED** [review]
LGTM @sagarkharbe 🎉 

For future reference, it's good to give your branch a more descriptive name and start it with a scope. See `#3` just below [here in the docs](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-setup-freecodecamp-locally.md#making-changes-locally)

**Comment by Sky020** [comment]
I am fine with the code, as it passes. However, I am clarifying that it is OK that it does not match the seed code?

**Comment by moT01** [comment]
I didn't look close enough. I thought log was for debugging when @sagarkharbe was making his solution or something. It's really probably fine, but we might as well add it back in I think. Sorry about that @sagarkharbe, if you don't mind adding that back.

Nice catch @Sky020.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/use-breadth-first-search-in-a-binary-search-tree.english.md [hunk 2]
```diff
@@ -106,7 +106,41 @@ BinarySearchTree.prototype = Object.assign(
 <section id='solution'>
 
 ```js
-// solution required
+function Node(value) {
```
Comment (moT01) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/use-breadth-first-search-in-a-binary-search-tree.english.md:109]:
```suggestion
var displayTree = tree => console.log(JSON.stringify(tree, null, 2));
function Node(value) {
```

**Review by moT01 — APPROVED** [review]
LGTM :tada:

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Comment by moT01** [comment]
Congratulations on your first contribution to freeCodeCamp @sagarkharbe 🎉

We look forward to future contributions 🎉

**Pull request merged**